xbmc/epg/EpgInfoTag.cpp

```diff
@@ -51,7 +51,7 @@ CEpgInfoTag::CEpgInfoTag(void) :
     m_iSeriesNumber(0),
     m_iEpisodeNumber(0),
     m_iEpisodePart(0),
-    m_iUniqueBroadcastID(-1),
+    m_iUniqueBroadcastID(0),
```

Comment:
Is is possible that there are EPG entries stored in the database that have `-1` here? If so you'd have to add a migration to change `-1` to `0`.